docs/src/content/docs/stack/cloudflare.md

```diff
@@ -0,0 +1,162 @@
+---
+title: Cloudflare Workers
+description: Serverless runtime and deployment platform for HONC applications
+---
+
+[Cloudflare Workers](https://workers.cloudflare.com/) provide a serverless runtime and deployment platform for building and scaling applications across Cloudflare's global network.
+Cloudflare handles the infrastructure and enables automatic scaling with exceptional performance.
+
+In the HONC stack, Cloudflare Workers serve as a serverless runtime for backend applications, enabling serverless APIs, background jobs, scheduled tasks, and more.
+
+## Worker Configuration
+[The configuration](https://developers.cloudflare.com/workers/wrangler/configuration/) for Cloudflare Workers in the HONC stack is stored in `wrangler.toml` (or alternatively `wrangler.json`/`wrangler.jsonc`). This configuration file defines settings for both development and production environments, including deployment configurations for different environments and bindings to Cloudflare services.
+
+```toml
+name = "honc-d1-template"
+compatibility_date = "2025-02-04"
+compatibility_flags = [ "nodejs_compat" ]
+```
+
+## Local Development with Wrangler
+Cloudflare supports two main tools for [local development](https://developers.cloudflare.com/workers/development-testing/#local-development): Wrangler and Vite. Honc templates use Wrangler as the primary local development tool. Wrangler, which internally uses Miniflare, is particularly well-suited for backend worker applications. 
+
+To start the local development server, HONC templates include a preconfigured script in the project's `package.json`:
+
+```json, title="package.json"
+{
+  "scripts": {
+    "dev": "wrangler dev src/index.ts"
+  }
+}
+```
+
+### Secrets
+For managing secrets during local development, create a `.dev.vars` file in the project root. This file contains the required environment variables for the application. The `.dev.vars` file follows a `dotenv` structure with `KEY="value"` pairs.
+
+```env, title=".dev.vars"
+DATABASE_URL="your-db-url-here"
+``` 
+
+### [Bindings](#cloudflare-worker-bindings)
+Local instances of Bindings (D1, KV, R2, etc.) are automatically created when running `wrangler dev`.
+
+Use the wrangler CLI to [populate data](https://developers.cloudflare.com/workers/development-testing/local-data/#populating-local-resources-with-data) to local generated developer primitives (D1, KV, R2, etc.)
+
+Additionally, for testing against Cloudflare services running on the Cloudflare network rather than emulated ones, Wrangler supports [remote bindings](https://developers.cloudflare.com/workers/development-testing/#remote-bindings)  with the `--remote` flag (still in beta):
```

Comment:
so code sample below uses the experimental remote bindings - should probably update the `--remote` sample here to say `--x-remote-bindings` to be coherent with the block below